frontend/src/metabase/public/components/EmbedFrame.jsx

```diff
@@ -61,9 +61,7 @@ export default class EmbedFrame extends Component {
     } = this.props;
     const { innerScroll } = this.state;
 
-    const showFooter =
-      !MetabaseSettings.hideEmbedBranding() ||
-      (actionButtons && actionButtons.length > 0);
+    const showFooter = !MetabaseSettings.hideEmbedBranding() || actionButtons;
```

Comment:
@robdaemon does this bring back the "powered by Metabase" section?

Reply:
Not that I know of, this condition is effectively the same?

Reply:
Can you please make sure? I believe the footer contains "powered by Metabase", and that's why we hide it. If `hideEmbedBranding` is true, we must not show that attribution link

Reply:
Yep, the desired behavior is still maintained with this change: https://github.com/metabase/metabase/blob/master/frontend/src/metabase/public/components/EmbedFrame.jsx#L116-L123

Reply:
cool, thanks for checking!